src/chi-vue/.eslintrc.js

```diff
@@ -22,5 +22,6 @@ module.exports = {
         endOfLine: 'auto',
       },
     ],
+    '@typescript-eslint/no-explicit-any': ['off'],
```

Comment:
@jllr @dani-cl-madrid Thoughts on disabling this rule? In this project there very few usages of the type any and almost everything is strongly typed. I would remove this

Reply:
yea but if you want to use Store later you'll need to use many `any` and then you'll have so many eslint ignore lines in the code which is not very convenient